setup/start.sh

```diff
@@ -35,7 +35,8 @@ if [ -f /etc/mailinabox.conf ]; then
 
 	# Load the old .conf file to get existing configuration options loaded
 	# into variables with a DEFAULT_ prefix.
-	cat /etc/mailinabox.conf | sed s/^/DEFAULT_/ > /tmp/mailinabox.prev.conf
+	cp /etc/mailinabox.conf /tmp/mailinabox.prev.conf
```

Comment:
Doing this in two separate operations is weird.
```suggestion
	sed 's/^/DEFAULT_/' /etc/mailinabox.conf > /tmp/mailinabox.prev.conf
```

Reply:
rm next line as well of course

Reply:
Did you test this? If so, great change and we can also do this below.

Reply:
I'll get a chance to test properly this evening. It's a standard syntax I've used in hundreds of scripts though. :)

Reply:
Tested. Same good end-result. :)

Reply:
Thank you! I will make the change here and at the two places below.